noetic/distribution.yaml

```diff
@@ -383,31 +383,6 @@ repositories:
       url: https://github.com/astuff/automotive_autonomy_msgs.git
       version: master
     status: maintained
-  autoware_msgs:
-    doc:
-      type: git
-      url: https://github.com/autoware-ai/messages.git
-      version: master
-    release:
-      packages:
-      - autoware_can_msgs
-      - autoware_config_msgs
-      - autoware_external_msgs
-      - autoware_lanelet2_msgs
-      - autoware_map_msgs
-      - autoware_msgs
-      - autoware_system_msgs
-      - tablet_socket_msgs
-      - vector_map_msgs
-      tags:
-        release: release/noetic/{package}/{version}
-      url: https://github.com/autoware-ai/messages-release.git
-      version: 1.14.0-1
-    source:
-      type: git
-      url: https://github.com/autoware-ai/messages.git
-      version: master
-    status: maintained
```

Comment:
It looks like there are downstream dependencies of at least one of these packages already in the rosdistro index.

Might I suggest, as an alternative to pulling the packages out of the index, you flip the `status` field to `end-of-life` and leave the packages alone?

Reply:
Thank you for your suggestion. I've flipped it instead. https://github.com/ros/rosdistro/pull/41122/commits/f293a01c54fb579c4244854e7ee528bab2034295